governance/proposals/up/008-apebond-liquidity.js

```diff
@@ -0,0 +1,42 @@
+const { ethers } = require('hardhat')
+const ERC20_ABI = require('@unlock-protocol/hardhat-helpers/dist/ABIs/erc20.json')
+
+module.exports = async () => {
+  const recipientAddress = '0xa7865ECE6DAB013E7131983b943c2c75D7Fa0D1F'
+  const upTokenAddress = '0xaC27fa800955849d6D17cC8952Ba9dD6EAA66187'
+  const initialTransferAmount = ethers.parseUnits('6092920', 'ether')
```

Comment:
Can you not use `ether` here but the numver of decimals? This would be less confusing!